### PR TITLE
feat: project creation activity

### PR DESCRIPTION
**Pull request by sk-yagi**
Resolve https://app.asana.com/0/1203842063837585/1204097242463663/f
In page details
Screen Shot 2023-03-11 at 15 41 21

In profile page:
Screen Shot 2023-03-11 at 16 52 21

### CODE REVIEW

File: modules/next-backend/logic/getProjectCreation.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// @sk-yagi: We should change this to something more Creator-related...
```
Comment (sk-kitsune) [inline modules/next-backend/logic/getProjectCreation.ts:1]: I think we should keep using the term "project" in our code base.

"Creator" can be misleading.

Comment (sk-kitsune) [reply modules/next-backend/logic/getProjectCreation.ts:1]: only when displaying, we use the term "Creator"

Comment (sk-kitsune) [reply modules/next-backend/logic/getProjectCreation.ts:1]: imagine how to explain these naming conventions to a new dev 👁️ 👄 👁️

Comment (sk-shishi) [reply modules/next-backend/logic/getProjectCreation.ts:1]: - Which actor is this?
- Oh, it's "creator owner", simple.

File: modules/next-backend/logic/getAllActivities.ts [hunk 2]
```diff
@@ -210,6 +212,25 @@ export async function getAllActivities(
...
+        DISTINCT ON (pd.project_id) pd.project_id,
+        p.owner_address AS created_by,
+        o.tx_id AS created_tx,
+        o.id AS output_id,
+        b.time,
+        6 AS action,
+        pd.sponsorship_amount
+      FROM
+        chain.project_detail pd
+        INNER JOIN chain.project p ON p.project_id = pd.project_id
```
Comment (sk-shishi) [inline modules/next-backend/logic/getAllActivities.ts:228]: Swap the join

File: modules/next-backend/logic/getAllActivities.ts [hunk 3]
```diff
@@ -210,6 +212,25 @@ export async function getAllActivities(
...
+        INNER JOIN ipfs.project_info pi ON pi.cid = pd.information_cid
+        INNER JOIN chain.output o ON o.id = pd.id
+        INNER JOIN chain.block b ON o.created_slot = b.slot
+      WHERE EXISTS (SELECT FROM all_referenced_projects a WHERE a.project_id = pd.project_id)
+      ORDER BY pd.project_id, pd.id
```
Comment (sk-shishi) [inline modules/next-backend/logic/getAllActivities.ts:233]: Be explicit about ASC vs DESC

File: modules/next-backend/logic/getProjectCreation.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  | {
+      error: null;
+      createdAt: UnixTimestamp;
+      sponsorshipAmount: LovelaceAmount | null; // undefined means no sponsor
+    }
+  | {
+      error: "no-project-found";
+    };
+
+export async function getProjectCreation(
```
Comment (sk-shishi) [inline modules/next-backend/logic/getProjectCreation.ts:24]:
```suggestion
export async function getProjectCreationActivity(
```
Rename this file too.
And don't forget to rebase.

**Review by sk-shishi — APPROVED** [review]

**Pull request merged**